eng/pipelines/coreclr/templates/build-job.yml

```diff
@@ -202,6 +202,10 @@ jobs:
       - script: $(Build.SourcesDirectory)/src/coreclr/build-runtime$(scriptExt) $(buildConfig) $(archType) -hostarch x64 $(osArg) -ci $(compilerArg) -component crosscomponents -cmakeargs "-DCLR_CROSS_COMPONENTS_BUILD=1" $(officialBuildIdArg) $(clrRuntimePortableBuildArg)
         displayName: Build CoreCLR Cross-Arch Tools (Tools that run on x64 targeting x86)
 
+    - ${{ if and(eq(parameters.osGroup, 'windows'), eq(parameters.archType, 'x86')) }}:
```

Comment:
This is reverting https://github.com/dotnet/runtime/pull/85947/files#diff-0605564a3ca1b4b1eca0cc0450b7628c0d027841259052a35825bd2e035484dfL205-L207 with a slight modification: We used to build the linux-arm targeting DAC in Windows arm build that is gone. We build it in Windows x86 build now.